com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -693,6 +784,67 @@ static void ReadFromShaderCoeffsL2(ref SphericalHarmonicsL2 sh, NativeArray<floa
             sh[0, 8] = shaderCoeffsL2[offset + 12]; sh[1, 8] = shaderCoeffsL2[offset + 13]; sh[2, 8] = shaderCoeffsL2[offset + 14];
         }
 
+        static void ReadFullFromShaderCoeffsL0L1L2(ref SphericalHarmonicsL2 sh, NativeArray<float> shL0L1Data, NativeArray<float> shL2Data, int probeIdx)
+        {
+            ReadFromShaderCoeffsL0L1(ref sh, shL0L1Data, probeIdx * ProbeVolumeAsset.kL0L1ScalarCoefficientsCount);
+            ReadFromShaderCoeffsL2(ref sh, shL2Data, probeIdx * ProbeVolumeAsset.kL2ScalarCoefficientsCount);
+
+        }
+
+        static void ExtractBakingCells()
+        {
+            List<BakingCell> outCells = new List<BakingCell>();
```

Comment:
this array is useless

Reply:
Good catch! Again another leftover of an old version :P This thing wen through a million iterations